fixtures/loader.go

```diff
@@ -360,23 +360,27 @@ func (f *Loader) buildInsertQuery(ctx *loadContext, t string, rows table) (strin
 	for i, row := range rows {
 		dbValuesRow := make([]string, len(fields))
 		for k, name := range fields {
-			value, _ := row[name]
-			// resolve references
-			if stringValue, ok := value.(string); ok {
-				if len(stringValue) > 0 && stringValue[0] == '$' {
-					var err error
-					dbValuesRow[k], err = f.resolveExpression(stringValue, ctx)
-					if err != nil {
-						return "", err
+			if value, present := row[name]; present {
+				// resolve references
+				if stringValue, ok := value.(string); ok {
+					if len(stringValue) > 0 && stringValue[0] == '$' {
+						var err error
+						dbValuesRow[k], err = f.resolveExpression(stringValue, ctx)
+						if err != nil {
+							return "", err
+						}
+						continue
 					}
-					continue
 				}
+				dbValue, err := toDbValue(value)
+				if err != nil {
+					return "", fmt.Errorf("unable to process %s value (row %d of %s): %s", name, i, t, err.Error())
+				}
+				dbValuesRow[k] = dbValue
+			} else {
+				// default is a PostgreSQL keyword
+				dbValuesRow[k] = "default"
```

Comment:
Isn't there is already too much nesting here? I think it will be more readable with guard clause.

```
for k, name := range fields {
	value, present := row[name]
	if !present {
		dbValuesRow[k] = "default" // default is a PostgreSQL keyword
		continue
	}
	
	// resolve references
	if stringValue, ok := value.(string); ok {
		...
	}
}
```

Reply:
Yes, good point. Thanks.

Reply:
Fixed.